soc/andestech/ae350/linker.ld

```diff
@@ -423,8 +423,8 @@ __rom_region_end = LOADADDR(.last_section) + SIZEOF(.last_section);
 SECTION_PROLOGUE(rom_mpu_padding,(NOLOAD),)
 {
 	MPU_ALIGN(__rom_region_size);
-	__rom_region_end = .;
 } GROUP_LINK_IN(ROMABLE_REGION)
+__rom_region_end = LOADADDR(rom_mpu_padding) + SIZEOF(rom_mpu_padding);
```

Comment:
It seems comment on line 403 is being contradicted by this change - maybe update the comment as well?

Reply:
Thanks for your suggestion. 

I think the comment on line 403 still applies to this change.
`__rom_region_end`  is still at the end of the linker script, but the last section is `rom_mpu_padding` instead of  `.last_section`.

I moved this comment to the assignment of `__rom_region_end` to avoid confusion.
```
/* Because ROMABLE_REGION != RAMABLE_REGION in XIP-system, it is valid
 * to set __rom_region_end symbol at the end of linker script and
 * doesn't mistakenly contain the RAMABLE_REGION in it.
 */
```